TestFoundation/TestNSJSONSerialization.swift

```diff
@@ -956,11 +956,524 @@ extension TestNSJSONSerialization {
         }
     }
     
-    private func removeTestFile(_ location: String) {
+    fileprivate func removeTestFile(_ location: String) {
         do {
             try FileManager.default.removeItem(atPath: location)
         } catch _ {
             
         }
     }
 }
+
+//MARK: - JSONObjectWithStreams
```

Comment:
Ok, I appreciate the extended test coverage here - but there has got to be a better way than copy/pasting all the existing test cases and just passing a stream with the same content.

Can we perhaps make a helper function which tests both data and stream versions and modify the existing tests?